### PR TITLE
Add option to persist card form data

### PR DESCRIPTION
**Pull request by crookedneighbor**
### Summary

Adds option to persist card data after tokenization. This can be useful for when the transaction fails and the customer needs to verify the card they input was correct.

### Checklist

- [x] Added a changelog entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ unreleased
 - Enable Google Pay support
 - Fix issue where non utf-8 encoded sites would show strange characters for card placeholders
 - Fix issue where card fields could not be focused by clicking on the corresponding label
+- Allow card form to not be cleared after succesful tokenization with `card.persistCardDataAfterTokenization`
```
Comment (crookedneighbor) [inline CHANGELOG.md]: How do we feel about the naming for this? Too verbose? Do we like the word `tokenization` in here? We don't use it in the Drop-in docs, even though that is what is happening.

Comment (idontusenumbers) [reply CHANGELOG.md]: Maybe something to the effect of 'allowEditingOfSavedPaymentMethods()' it's just as verbose but might be easier to understand.

**Comment by crookedneighbor** [comment]
:+1: Except that we intentionally don't use the word tokenization in Drop-in. (hence why we have `requestPaymentMethod`)

**Comment by crookedneighbor** [comment]
Also, just noticed that I used the word tokenization in my original version of this :shame:

**Comment by crookedneighbor** [comment]
@quinnjn and I talked about it and we're just going to go with `clearCardFieldsAfterTokenization`

**Pull request merged**